lib/themes/dosomething/paraneue_dosomething/js/utilities/Analytics.js

```diff
@@ -40,6 +40,14 @@ function subscribeEvents() {
     const label = args !== null ? '#' + args.attr('id') : '';
     ga('send', 'event', 'Modal', 'Close', label);
   });
+
+  // Signup button on pitch page
+  const urlParts = window.location.pathname.split('/');
+  if (urlParts[urlParts.length - 1] === 'pitch') {
+    $('#dosomething-signup-form-primary').on('submit', () => {
```

Comment:
Good catch! `link--campaign-signup-login` is the other ID.

Potentially better option, listen for the button submit under `.header__signup`